ci/devbox/build.sh

```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+if ! type packer; then
+  echo "packer command not found. Please install from https://packer.io/" >&2
+  exit 1
+fi
```

Comment:
Very nice to see this. :) Much better than downloading a specific one without asking.